### PR TITLE
Added My Python Function

### PR DESCRIPTION
**Pull request by abughurab16**
please tell me what you think about the code?

### CODE REVIEW

File: .vim/coc-settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (AllanRegush) [inline .vim/coc-settings.json]: I don't think we need the editor config.

**Review by AllanRegush — COMMENTED** [review]
Code Looks Great :+1: Thank you.

File: .vim/coc-settings.json (hunk 1 above)
Comment (abughurab16) [reply .vim/coc-settings.json]: oops i didnt notice that sorry

**Review by schmelto — COMMENTED** [review]
Mya you want to adapt clean code principles to the variables names eq. W -> inputCharacter

**Comment by AllanRegush** [comment]
@schmelto Thank you for reviewing. :nerd_face:  But this was part of the requirements. #3

**Review by AllanRegush — APPROVED** [review]
Looks Good Thank you.

**Pull request merged**